tests/system/conftest.py

```diff
@@ -84,12 +92,34 @@ def bigquery_dataset(
 
 
 @pytest.fixture(scope="session", autouse=True)
-def bigquery_empty_table(bigquery_dataset, bigquery_client, bigquery_schema):
+def bigquery_dml_dataset(bigquery_client: bigquery.Client):
+    project_id = bigquery_client.project
+    dataset_id = "test_pybigquery_dml"
+    dataset = bigquery.Dataset(f"{project_id}.{dataset_id}")
+    dataset = bigquery_client.create_dataset(dataset, exists_ok=True)
+    return dataset_id
+
+
+@pytest.fixture(scope="session", autouse=True)
+def bigquery_empty_table(
+    bigquery_dataset: str,
+    bigquery_dml_dataset: str,
+    bigquery_client: bigquery.Client,
+    bigquery_schema: List[bigquery.SchemaField],
+):
     project_id = bigquery_client.project
-    dataset_id = bigquery_dataset
-    table_id = f"{project_id}.{dataset_id}.sample_dml"
+    # Cleanup the sample_dml table, if it exists.
+    old_table_id = f"{project_id}.{bigquery_dataset}.sample_dml"
+    bigquery_client.delete_table(old_table_id, not_found_ok=True)
+    # Create new table in its own dataset.
+    dataset_id = bigquery_dml_dataset
+    table_id = f"{project_id}.{dataset_id}.sample_dml_{temp_suffix()}"
     empty_table = bigquery.Table(table_id, schema=bigquery_schema)
-    bigquery_client.create_table(empty_table, exists_ok=True)
+    # Add table expiration in case cleanup fails.
+    empty_table.expires = datetime.datetime.utcnow() + datetime.timedelta(days=1)
```

Comment:
Rather than doing this to tables individually, perhaps just set the default expiration to 1d on the `bigquery_dml_dataset`?

Reply:
Good call. Done